### ISSUE TITLE
Listed in the NeoLine mobile app gallery

### ISSUE THREAD

**Issue by mike-petrov** [nspcc-dev/panel-fs-neo-org#224]
<img src="https://github.com/user-attachments/assets/7d4580b5-ab17-45c6-b153-953bb6a1a3d6" width=300 />
<img src="https://github.com/user-attachments/assets/1d8040ad-d8d4-45a1-a4cc-c774837a8b2b" width=300 />
<img src="https://github.com/user-attachments/assets/e493ab74-67f9-4726-bd84-a68d82e58227" width=300 />
<img src="https://github.com/user-attachments/assets/43975df3-6dfe-4e47-9839-b889c1d5ef06" width=300 />

**Comment by roman-khimov** [comment]
Don't we need to add support for it in some way before submitting anything?

**Comment by mike-petrov** [comment]
Yes, we should add support, the question is do we want to do it?

**Comment by mike-petrov** [comment]
done, now we are waiting for updates

<img width="382" height="324" alt="Image" src="https://github.com/user-attachments/assets/5da1a8d9-8c9d-42e6-bae2-5fec1ac018fc" />

**Comment by roman-khimov** [comment]
Any news here?

**Comment by mike-petrov** [comment]
@roman-khimov no news yet. Their apps haven’t been updated for quite a while, so it’s possible this form simply isn’t being reviewed.
